addons/test_mail/tests/test_mail_message.py

```diff
@@ -170,6 +170,24 @@ def test_mail_message_values_fromto_long_name(self):
         self.assertEqual(msg.reply_to, formataddr((test_record.name, reply_to_email)),
                          'Reply-To: use recordname as name in format if recordname + company > 78 chars')
 
+        # no record_name: keep company_name in formatting if ok
+        test_record.write({'name': ''})
```

Comment:
The change is explained in the commit message of 4573ca0c83eb63785016f4389a5157efb21fa9a4.
> Since `display_name` should never be Falsy anymore, remove part of the
test_mail_message_values_fromto_long_name that covers the
Falsy `display_name` case.

Reply:
You should have updated the test then maybe, in order to see the change, not removed it ^^ . Do you mean people could have received ``"YourCompany (32, project.task)" <tata@yoyo.fr>`` ?

Reply:
They will receive: `"YourCompany <model_name>.<id>"`. And I didn't to want to test the same stuff twice (I tested the fallback in base and it doesn't make sense to test in mail too) ? The removed test part tested the case of `no record_name` (meaning display_name == False), which is no longer possible (expected if you fail your _compute_display_name). It is true that I could create a incorrect `_compute_display_name` to keep the test :thinking: .

Anyway, the test is back.